sbpy/activity/gas/core.py

```diff
@@ -27,14 +25,15 @@
     import scipy
     from scipy import special
     from scipy.integrate import quad, dblquad, romberg
-    from scipy.interpolate import CubicSpline
+    from scipy.interpolate import CubicSpline, PPoly
 except ImportError:
     scipy = None
```

Comment:
Do you still need to define variable `scipy` and `PPoly` if scipy is not imported, if you use `requires` decorator to test the import of scipy?

Of course there is no problem to define them, just curious.

Reply:
PPoly is used in the type annotations.  I can probably remove scipy.